perf.groovy

```diff
@@ -37,3 +55,61 @@ def projectFolder = Utilities.getFolderName(project) + '/' + Utilities.getFolder
         }
     }
 }
+
+// Create the Linux/OSX/CentOS coreclr test leg for debug and release and each scenario
+[true, false].each { isPR ->
+    ['Ubuntu'].each { os ->
+        def osGroup = getOSGroup(os)
+        def architecture = 'x64'
+        def configuration = 'Release'
+        def newJob = job(Utilities.getFullJobName(project, "perf_${osGroup}", isPR)) {
+            steps {
+                // Set up the copies
+                // Coreclr build containing the tests and mscorlib
+                shell('curl http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/master/job/release_windows_nt/lastSuccessfulBuild/artifact/bin/tests/tests.zip -o tests.zip')
```

Comment:
I do not think this brings anything to the table besides making the system harder to test. That being said, we hopefully never change configurations of jobs manually and in my opinion should disable that option as it changes the job. This would not only be difficult to track over time, but difficult to track through automation. It has the added bonus of being prone to user error as it globally changes the job definition with no expectation of configuring it back to what it was until generation time.

On the flip side a hoisted script is testable before check-in either manually or through automated testing. It is cleaner, easier to configure and normally easier to find. These all translate to easier to maintain. 

/CC @adiaaida @mmitche